src/Makefile.am

```diff
@@ -246,6 +246,22 @@ libeos_shell_js_la_CPPFLAGS =		\
 
 ########################################
 
+# Set up headers for libeos-shell-fx
+eos_shell_fxincludedir = $(includedir)
+eos_shell_fxinclude_HEADERS = wobbly-effect.h
+
+eos_shell_fx_cppflags = $(AM_CPPFLAGS) $(EOS_SHELL_FX_CFLAGS) -std=c++11 -Werror -Wall -Wextra -Wno-error=unused-parameter -Wno-error=deprecated-declarations
```

Comment:
The philosophy here is mainly that they don't hurt and they catch bad things before they sneak into production code. I don't believe they're enabled for the rest of the project (nor would we want to do that, shell already compiles with warnings in a few places and now isn't the right time to go fixing all of those).

I suppose the tradeoff is that you have a slightly higher maintenance burden in terms of dealing with two different sets of warning flags. If there's a reason why that burden is high enough to justify having the same flags, then I can remove the specification, otherwise I think it should stay.

That said, I believe -std=c++11 is required for some of the functionality this code uses, or at least some of the stuff that's in libwobbly's headers. If we really cared, I could create a c++98 compatible interface, but since c++11 has been around for what feels like forever now, I'm not sure if there's really any benefit in doing so.

(Caveat: Back when I wrote this feature I was a much more of a C++ fanatic than I am now. Not that I'm jaded by the language, just that for whatever reason, I've had a lot more exposure to C, Python and JS lately).